crates/core/tedge/src/bridge/c8y.rs

```diff
@@ -87,6 +99,34 @@ impl From<BridgeConfigC8yParams> for BridgeConfig {
             .collect::<Vec<String>>();
         topics.extend(templates_set);
 
+        // SmartRest1 (to support customers with existing solutions based on SmartRest 1)
+        // Only add the topics if at least 1 template is defined
+        if !smartrest_one_templates.0.is_empty() {
+            topics.extend([
+                format!(r#"s/ul/# out 2 {topic_prefix}/ """#),
```

Comment:
```suggestion
                format!(r#"s/ul/# out 1 {topic_prefix}/ """#),
```
In a recent PR, we switched all those QoS values to 1, as Cumulocity doesn't support 2. So, you may wanna change all the following ones as well to `1`.

Reply:
This PR and #3201 will make a conflict to each other. The one gets merged faster will fix ;)